library/std/src/os/unix/process.rs

```diff
@@ -24,26 +24,26 @@ pub trait CommandExt: Sealed {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn uid(
         &mut self,
-        #[cfg(not(target_os = "vxworks"))] id: u32,
-        #[cfg(target_os = "vxworks")] id: u16,
+        #[cfg(not(any(target_os = "vxworks", target_os = "horizon")))] id: u32,
+        #[cfg(any(target_os = "vxworks", target_os = "horizon"))] id: u16,
```

Comment:
I wonder why these don't use the libc types. I know this PR is already merged, but maybe we should make that change?

Reply:
good point, I wonder if that would resolve these `cfg`... Maybe I can roll this into #14